app/utilities/im_processing.py

```diff
@@ -27,4 +28,7 @@ def load_img(path, size: Union[tuple[int] | float] = None):
         new_size = tuple(int(length * size) for length in img.size)
         return ImageTk.PhotoImage(img.resize(new_size))
 
-    raise ValueError("Expected size to be either a tuble of ints, a float or None.")
+    
+#Grammar correction tuble to tuple 
```

Comment:
You may add comments explaining what the code does, but you don't need to add comments directly in the code saying what you changed (these comments will persist and might be confusing later after this PR is closed). You can explain your changes in the Pull Request.
Please remove this one.

Reply:
Thanks, noted and working on it.